features/step_definitions/admin_steps.rb

```diff
@@ -340,3 +340,11 @@ def update_settings(settings)
   u = User.find_by_login(user)
   u.update_attribute(:banned, false)
 end
+
+When(/^the last login information has been updated$/) do
+  User.update_last_login
+end
+
+Then(/^I should see the last login time/) do
+  step('I should not see "No logins recorded"')
```

Comment:
Here I think you want something a bit more specific and also this step doesn't actually match completely.  

```
Then /^I should see the last login time for "([^"]*)"$/ do |user|
  u = User.find_by_login(user)
  assert u.last_active_at.match(//) # not sure what the format is but eg the current year or something
  step %{I should see "#{u.last_active_at}"}
end
```

which will actually make sure that the value is what you expect and that it is displayed.